src/namespace.rs

```diff
@@ -0,0 +1,167 @@
+use std::convert::Into;
+use std::env::current_dir;
+use std::env::set_current_dir;
+use std::fs;
+use std::fs::File;
+use std::io;
+use std::io::BufRead;
+use std::io::BufReader;
+use std::os::fd::AsFd;
+use std::os::fd::AsRawFd;
```

Comment:
Let's switch to `use std::os::unix::io::AsFd` and `use std::os::unix::io::AsRawFd`, to make CI happy:
```suggestion
use std::os::unix::io::AsFd as _;
use std::os::unix::io::AsRawFd as _;
```